java/dev/checkstyle/checkstyle.xml

```diff
@@ -223,14 +224,8 @@
         <module name="JavadocMethod">
             <property name="scope" value="public"/>
             <property name="allowMissingParamTags" value="true"/>
-            <property name="allowMissingThrowsTags" value="true"/>
             <property name="allowMissingReturnTag" value="true"/>
-            <property name="minLineCount" value="2"/>
             <property name="allowedAnnotations" value="Override, Test"/>
-            <property name="allowThrowsTagsForSubclasses" value="true"/>
-            <!-- This seems partially broken under JDK >= 9. -->
-            <property name="suppressLoadErrors" value="true"/>
-            <property name="ignoreMethodNamesRegex" value="main"/>
```

Comment:
Are these checks no longer supported?

Reply:
It showed up errors when running with those configs. I didn't log them, but I can reproduce the errors and post here?

Reply:
Yes, let's just take a look and make sure - I don't care too much either way but perhaps some of these will still be useful even after google-java-format (since many of these seem more like lint things)

Reply:
Yes I agree, let me log one by one here.

Reply:
Listing compile errors for the removed attributes

1. `<property name="allowMissingThrowsTags" value="true"/>`

```bash
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.0:check (validate) on project arrow-maven-plugins: Failed during checkstyle configuration: cannot initialize module TreeWalker - cannot initialize module JavadocMethod - Property 'allowMissingThrowsTags' does not exist, please check the documentation -> [Help 1]
```

2. `<property name="minLineCount" value="2"/>`

```bash
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.0:check (validate) on project arrow-maven-plugins: Failed during checkstyle configuration: cannot initialize module TreeWalker - cannot initialize module JavadocMethod - Property 'minLineCount' does not exist, please check the documentation -> [Help 1]
```

3. `<property name="allowThrowsTagsForSubclasses" value="true"/>`

```bash
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.0:check (validate) on project arrow-maven-plugins: Failed during checkstyle configuration: cannot initialize module TreeWalker - cannot initialize module JavadocMethod - Property 'allowThrowsTagsForSubclasses' does not exist, please check the documentation -> [Help 1]
```

4. `<property name="suppressLoadErrors" value="true"/>`

```bash
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.0:check (validate) on project arrow-maven-plugins: Failed during checkstyle configuration: cannot initialize module TreeWalker - cannot initialize module JavadocMethod - Property 'suppressLoadErrors' does not exist, please check the documentation -> [Help 1]
```

5. `<property name="ignoreMethodNamesRegex" value="main"/>`

```bash
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.0:check (validate) on project arrow-maven-plugins: Failed during checkstyle configuration: cannot initialize module TreeWalker - cannot initialize module JavadocMethod - Property 'ignoreMethodNamesRegex' does not exist, please check the documentation -> [Help 1]
```

Reply:
@lidavidm these are the errors I saw when each of these properties were used.

Reply:
`https://github.com/checkstyle/checkstyle/issues/7329`

- allowMissingThrowsTags, allowThrowsTagsForSubclasses, suppressLoadErrors are now basically the default

`https://github.com/checkstyle/checkstyle/issues/6703`

- minLineCount and ignoreMethodNamesRegex were moved to a new check called `MissingJavadocMethodCheck`, can you configure that to behave like our old one?

Reply:
Ah I see. Let me check.

Reply:
@lidavidm I added the `MissingJavadocMethodCheck`

Latest version: https://checkstyle.sourceforge.io/version/8.29/config_javadoc.html#JavadocMethod

Previois version: https://checkstyle.sourceforge.io/version/8.19/config_javadoc.html#JavadocMethod

Reply:
> allowMissingThrowsTags, allowThrowsTagsForSubclasses, suppressLoadErrors are now basically the default

These are not explicity properties in `8.29` but were in `8.19` but I couldn't find whether they are the defaults in `8.29`.